Scroller/EditableWrapper.js

```diff
@@ -213,14 +239,18 @@ const EditableWrapper = (props) => {
 
 	const handleKeyDown = useCallback((ev) => {
 		const {keyCode, target} = ev;
-		const {selectedItem} = mutableRef.current;
-		if (is('enter', keyCode)) {
+		const {selectedItem, spotlightId} = mutableRef.current;
+		const targetItemNode = findItemNode(target);
+
+		if (is('enter', keyCode) && target.getAttribute('role') !== 'button') {
```

Comment:
Why do we need to check `target.getAttribute('role') !== 'button'` here?

Reply:
This is to allow button-type actions such as the remove button.

Reply:
Ah-ha!